src/runtime_src/core/edge/drm/zocl/zocl_sysfs.c

```diff
@@ -72,7 +72,7 @@ static ssize_t xclbinid_show(struct device *dev,
 	struct drm_zocl_dev *zdev = dev_get_drvdata(dev);
 	ssize_t size;
 
-	if (!zdev)
+	if (!zdev || !zdev->zdev_xclbin || !zdev->zdev_xclbin->zx_uuid)
 		return 0;
```

Comment:
If we print uuid_null, we don't really need any user space code change. Right?

Like this,
if (!zdev || !zdev->zdev_xclbin || !zdev->zdev_xclbin->zx_uuid) {
    size = sprintf(buf, "%pUb\n", &uuid_null);
} else {
    size = sprintf(buf, "%pUb\n", zdev->zdev_xclbin->zx_uuid);
}

Reply:
Thats a good suggestion, but we still need to make user space changes to check whether uuid is null or not to call xclOpenConext. xrt_uuid constructor change is also required as it gets used in multiple places.